### ISSUE TITLE
Accessibility: add searchbox role

### ISSUE THREAD

**Issue by GipHub123** [bithost-gmbh/ngx-mat-select-search#286]
**Describe the bug**

Accessibility related role-attribute is missing. 

Chrome's Lighthouse plugin gives following warning. 
"Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children."

```
<input _ngcontent-rt-xxx-c46="" 
	autocomplete="off" 
	class="mat-select-search-input mat-input-element ng-pristine ng-valid ng-touched" 
	ng-reflect-form="[object Object]" 
	type="text" 
	placeholder="Regions" 
	aria-label="Regions">
```

**Expected behavior**
Role -attribute is set

**Screenshots**

**Additional context**
 "ngx-mat-select-search": "^3.1.4",

Angular CLI: 11.0.2
Node: 10.18.1
OS: win32 x64

Angular: 11.0.2
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... platform-server, router
Ivy Workspace: Yes
Package Version

@angular-devkit/architect 0.1100.0
@angular-devkit/build-angular 0.1100.2
@angular-devkit/core 11.0.0
@angular-devkit/schematics 11.0.2
@angular/cdk 11.0.0
@angular/material 11.0.0
@angular/material-moment-adapter 11.0.0
@nguniversal/builders 11.0.0
@nguniversal/express-engine 11.0.0
@schematics/angular 11.0.2
@schematics/update 0.1100.2
rxjs 6.6.3
typescript 4.0.5

**Comment by macjohnny** [comment]
Thanks for reporting.
Would you like to fix this?

**Comment by macjohnny** [comment]
related: https://github.com/angular/components/issues/21354

**Comment by macjohnny** [comment]
I think we should add the `role="searchbox"` to the `ngx-mat-select-search` search input.
see https://github.com/angular/components/issues/21354#issuecomment-746008265

**Comment by macjohnny** [comment]
this is not something we can fix in ngx-mat-select-search, since the it will only be rendered in the dom if the select panel is open. therefore, according to https://github.com/angular/components/issues/21354, the testing tools should be updated to support ARIA 1.2, which resolves the issue.
